src/common/legacy_config_opts.h

```diff
@@ -515,6 +515,12 @@ OPTION(mds_damage_table_max_entries, OPT_INT)
 // Maximum increment for client writable range, counted by number of objects
 OPTION(mds_client_writeable_range_max_inc_objs, OPT_U32)
 
+//worm attribute default value
+OPTION(mds_worm_commit_period, OPT_U64)
+OPTION(mds_worm_retention_period, OPT_U64)
+OPTION(mds_worm_min_retention_period, OPT_U64)
+OPTION(mds_worm_max_retention_period, OPT_U64)
```

Comment:
no need to add options to  legacy_config_opts.h

Reply:
ok，fixed it，thanks